### ISSUE TITLE
Can't close frunk or trunk with aftermarket kits

### ISSUE THREAD

**Issue by iainswarts** [alandtse/tesla#325]
The `CoverEntity.CLOSE` is gated behind the `_car.powered_lift_gate` flag, in my case I've installed an aftermarket power trunk and frunk so I can no longer close either of them.

Can you add an option to override this somehow please?

_Originally posted by @iainswarts in https://github.com/alandtse/tesla/issues/307#issuecomment-1297991080_

**Comment by alexeiw123** [comment]
Ah good, this saves me creating the same issue!
Have rolled back to 2.4.4 for the time being so I can still open and close from my dashboard.

**Comment by alexeiw123** [comment]
> The `CoverEntity.CLOSE` is gated behind the `_car.powered_lift_gate` flag, in my case I've installed an aftermarket power trunk and frunk so I can no longer close either of them.
> 
> Can you add an option to override this somehow please?
> 
> _Originally posted by @iainswarts in [#307 (comment)](https://github.com/alandtse/tesla/issues/307#issuecomment-1297991080)_

You could use this in the meantime in your configuration.yaml to add a trunk and frunk cover that works with third party installations (hannshow for me)

I haven't tested this exact code as I'm using a teslamate binary sensor for my value_template, as it seems to update faster. I think this should still work however.

`cover:
  - platform: template
    covers:			  
  	<car>_trunk:
        device_class: door
        friendly_name: "<car> Trunk"
        unique_id: <car>_trunk_cover_api
        value_template: >-
          {{not(is_state_attr('sensor.<car>_vehicle_state_data_sensor','rt',0))}}
        open_cover:
          - service: tesla_custom.api
            data:
              command: ACTUATE_TRUNK
              parameters:
                path_vars:
                  vehicle_id: >-
                    {{ state_attr('binary_sensor.<car>_online_sensor', 'id') }}
                which_trunk: rear
                wake_if_asleep: true
        close_cover:
          - service: tesla_custom.api
            data:
              command: ACTUATE_TRUNK
              parameters:
                path_vars:
                  vehicle_id: >-
                    {{ state_attr('binary_sensor.<car>_online_sensor', 'id') }}
                which_trunk: rear
                wake_if_asleep: true
  	<car>_frunk:
        device_class: door
        friendly_name: "<car> Frunk"
        unique_id: <car>_bonnet_cover_api
        value_template: >-
          {{not(is_state_attr('sensor.<car>_vehicle_state_data_sensor','ft',0))}}
        open_cover:
          - service: tesla_custom.api
            data:
              command: ACTUATE_TRUNK
              parameters:
                path_vars:
                  vehicle_id: >-
                    {{ state_attr('binary_sensor.<car>_online_sensor', 'id') }}
                which_trunk: front
                wake_if_asleep: true
        close_cover:
          - service: tesla_custom.api
            data:
              command: ACTUATE_TRUNK
              parameters:
                path_vars:
                  vehicle_id: >-
                    {{ state_attr('binary_sensor.<car>_online_sensor', 'id') }}
                which_trunk: front
                wake_if_asleep: true`

**Comment by iainswarts** [comment]
I took a much simpler approach (using EVOffer parts), I just opened up `/config/custom_components/tesla_custom/cover.py` in the File Editor and edited line 105 and 145 to be

```
        return CoverEntityFeature.OPEN | CoverEntityFeature.CLOSE
```

and then restarted my HA and it worked just fine.

**Comment by alexeiw123** [comment]
Yes that's much more elegant! I guess that's the fix required for this integration really. Cover will work for me in the meantime :)
